crates/oxc_linter/src/snapshots/no_useless_spread.snap

```diff
@@ -637,13 +637,6 @@ source: crates/oxc_linter/src/tester.rs
    ╰────
   help: `foo.split` returns a new array. Spreading it into an array expression to create a new array is redundant.
 
-  ⚠ eslint-plugin-unicorn(no-useless-spread): Using a spread operator here creates a new array unnecessarily.
-   ╭─[no_useless_spread.tsx:1:2]
- 1 │ [...new Array(3)]
-   ·  ───
-   ╰────
-  help: `new Array(3)` returns a new array. Spreading it into an array expression to create a new array is redundant.
-
```

Comment:
i think we should still report this case - `unicorn/no-useless-spread` reports an error.
how about we just don't suggest a fix in this case

Reply:
I'm wondering if we should fix `[...new Array(3)]` to `new Array(3).fill()`